robottelo/bz_helpers.py

```diff
@@ -25,8 +41,11 @@ def get_wontfix_bugs(bugs=None):
     bugs = bugs or get_decorated_bugs()
     wontfixes = []
     for bug_id, data in bugs.items():
-        if data['bug_data']['resolution'] in ('WONTFIX', 'CANTFIX'):
-            wontfixes.append(bug_id)
+        bug_data = data.get('bug_data')
+        # when not authenticated, private bugs will have no bug data
+        if bug_data:
+            if bug_data['resolution'] in ('WONTFIX', 'CANTFIX'):
```

Comment:
nitpick: i'd rather use `and` instead of nested `if` statements (`if bug_data and bug_data in ...`)